manual_jb/content/analyses/equity_metric.md

```diff
@@ -0,0 +1,81 @@
+-----------------------------------------|-------|-------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+ `result_name` <sup>*</sup>              | `str` | Result name                   | Name of the result dataset.                                                                                                                                                          
+ `division_decision_column` <sup>*</sup> | `str` | Division decision column name | Column name of a binary variable associated with each household used to group it into two groups (e.g. low income vs non low income, minority vs non-minority, social vulnerability) 
+
+**Input datasets**
+
+ key name                               | type                                                                                                                | name               | description                                                            
+----------------------------------------|---------------------------------------------------------------------------------------------------------------------|--------------------|------------------------------------------------------------------------
+ `housing_unit_allocation` <sup>*</sup> | [`incore:housingUnitAllocation`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:housingUnitAllocation) | Housing allocation | A housing unit allocation dataset.                                     
+ `scarce_resource` <sup>*</sup>         | [`incore:scarceResource`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:scarceResource)               | Scarce resource    | Scarce resource dataset e.g. probability of service, return time, etc. 
+
+**Output datasets**
+
+ key name                     | type                                                                                              | parent key | name          | description                                                                                           
+------------------------------|---------------------------------------------------------------------------------------------------|------------|---------------|-------------------------------------------------------------------------------------------------------
+ `equity_metric` <sup>*</sup> | [`incore:equityMetric`](https://incore.ncsa.illinois.edu/semantics/api/types/incore:equityMetric) |            | Equity Metric | CSV file of equity metric, including Theil’s T Value, Between Zone Inequality, Within Zone Inequality 
+
+<small>(* required)</small>
+
+**Execution**
+
+code snippet:
+
+```
+    client = IncoreClient(pyglobals.INCORE_API_DEV_URL)
```

Comment:
We shouldn't refer to dev for creating the client in the snippet. We can either not show the client creation or use empty args.

Reply:
my bad. Just removed :-)